src/buildModels.ts

```diff
@@ -0,0 +1,76 @@
+import { sync as globSync } from 'globby'
+import { Sequelize } from 'sequelize'
+import chalk from 'chalk'
+
+const WARNED = new Set<string>()
+
+const buildModels = (
+  entitiesDirectory: string,
+  sequelizeInstance: Sequelize,
+) => {
+  const entityPaths = globSync(['*.js', '!*.spec.js'], {
+    cwd: entitiesDirectory,
+    absolute: true,
+  })
+
+  const deprecatedMap = new Map()
+
+  const entities = entityPaths.map((entityPath) => {
+    const entity = require(entityPath).default
+    if (Object.hasOwnProperty.call(entity, 'deprecatedNames')) {
+      for (const name of entity.deprecatedNames) {
+        deprecatedMap.set(name, entity.name)
+      }
+    }
+    return entity
+  })
+
+  const models = entities.reduce((models, entity) => {
```

Comment:
Let's give this function a name and export it.

Something like `buildSequelizeEntities` could do.

This function should have unit tests.